src/aks-preview/setup.py

```diff
@@ -8,7 +8,7 @@
 from codecs import open as open1
 from setuptools import setup, find_packages
 
-VERSION = "0.4.36"
+VERSION = "0.4.37"
```

Comment:
this will be a new release or bundle with others?

Reply:
i think we can ship a new release with this, it's a major feature